src/language-js/parse/postprocess/index.js

```diff
@@ -111,8 +117,22 @@ function postprocess(ast, options) {
             };
           }
         }
+
+        // In TypeScript, we can't distinguish `with`/`assert`
```

Comment:
Oh...

Reply:
Sorry, I have not been able to research this yet. If you know, please let me know.
Is this a problem with TypeScript Compiler? Or is it a typescript-estree issue? If it is a typescript-estree issue, is there any way to fix the parser side?

Reply:
I don't think typescript-estree cares about it, I think typescript treat them the same.